src/components/Book.js

```diff
@@ -1,30 +1,25 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
 const Book = () => {
-  const books = [
-    {
-      id: 1,
-      name: 'Spiderman 1998',
-      author: 'Peter packer',
-    },
-    {
-      id: 1,
-      name: 'Batman 1990',
-      author: 'Bruce Wayne',
-    },
-    {
-      id: 1,
-      name: 'Hulk 1978',
-      author: 'Bruce Jnr',
-    },
-  ];
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books.books);
+
   return (
-    <div>
-      {books.map((book) => (
-        <div className="book-section" key={book.name}>
-          <li>{book.name}</li>
-          <h2>{book.author}</h2>
-          <button className="del" type="button">Delete</button>
+    <div className="Book-C">
+      {books.map((item) => (
+        <div className="Single-B" key={item.item_id}>
+          <h3>{item.title}</h3>
+          <p>{item.author}</p>
+          <p>{item.category}</p>
+          <button
+            className="RemoveBook"
+            type="button"
+            onClick={() => dispatch(removeBook(item.item_id))}
+          >
+            Remove
+          </button>
         </div>
```

Comment:
- [ ] [Optional]👍 Great work! However, it's important to ensure that each component adheres to the project requirements. In your first milestone, there was [a specific requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md#:~:text=Books%20%2D%20the%20default,construction%22%20text%20only.) that this component should be responsible only for rendering each book. To align with the project requirements and maintain a well-structured codebase, please consider focusing this component on its designated task of rendering books. Keep up the good work! 👍📚